bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/container.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+
+import Service from './service';
```

Comment:
`PresentationUploaderService`

Reply:
We only prefix the service name when importing outside its pod.